[2023-12-21 14:00:05,067] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_python_weather_v01.fill 2023-12-21T13:00:00+00:00 [queued]>
[2023-12-21 14:00:05,128] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_python_weather_v01.fill 2023-12-21T13:00:00+00:00 [queued]>
[2023-12-21 14:00:05,129] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-12-21 14:00:05,137] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2023-12-21 14:00:05,138] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-12-21 14:00:05,177] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): fill> on 2023-12-21T13:00:00+00:00
[2023-12-21 14:00:05,190] {standard_task_runner.py:52} INFO - Started process 48029 to run task
[2023-12-21 14:00:05,204] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_python_weather_v01', 'fill', '2023-12-21T13:00:00+00:00', '--job-id', '224', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_python_weather.py', '--cfg-path', '/tmp/tmp19i9daxz', '--error-file', '/tmp/tmp6o0n65kl']
[2023-12-21 14:00:05,212] {standard_task_runner.py:77} INFO - Job 224: Subtask fill
[2023-12-21 14:00:05,642] {logging_mixin.py:104} INFO - Running <TaskInstance: dag_with_python_weather_v01.fill 2023-12-21T13:00:00+00:00 [running]> on host 5f3722f1fc3b
[2023-12-21 14:00:05,802] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ggoni
AIRFLOW_CTX_DAG_ID=dag_with_python_weather_v01
AIRFLOW_CTX_TASK_ID=fill
AIRFLOW_CTX_EXECUTION_DATE=2023-12-21T13:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-12-21T13:00:00+00:00
[2023-12-21 14:00:07,588] {python.py:118} INFO - Done. Returned value was: None
[2023-12-21 14:00:07,625] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=dag_with_python_weather_v01, task_id=fill, execution_date=20231221T130000, start_date=20231221T140005, end_date=20231221T140007
[2023-12-21 14:00:07,676] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-12-21 14:00:07,709] {local_task_job.py:146} INFO - Task exited with return code 0
